src/ILLink.RoslynAnalyzer/ILLink.RoslynAnalyzer.csproj

```diff
@@ -2,14 +2,33 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <LangVersion>8</LangVersion>
+    <LangVersion>latest</LangVersion>
+    <OutputType>Library</OutputType>
     <Nullable>enable</Nullable>
     <EnableDefaultEmbeddedResourceItems>false</EnableDefaultEmbeddedResourceItems>
+    <AutoGenerateAssemblyVersion>true</AutoGenerateAssemblyVersion>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+  </PropertyGroup>
+
+  <!-- Analyzer package properties -->
+  <PropertyGroup>
+    <IsPackable>true</IsPackable>
+    <PackageId>Microsoft.NET.ILLink.Analyzers</PackageId>
+    <Authors>Microsoft</Authors>
+    <RepositoryUrl>https://github.com/mono/linker</RepositoryUrl>
+    <Description>Description of the package</Description>
+    <NoPackageAnalysis>true</NoPackageAnalysis>
   </PropertyGroup>
 
   <ItemGroup>
     <EmbeddedResource Include="Resources.resx" GenerateSource="true" />
-    <PackageReference Include="microsoft.codeanalysis.csharp" Version="3.7.0" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="3.7.0" PrivateAssets="all" />
   </ItemGroup>
 
-</Project>
+  <Target Name="IncludeAnalyzerAssembliesInPkg" AfterTargets="AfterBuild">
```

Comment:
Might want to do `DependsOn=Build` and `BeforeTargets=Pack` here, since this needs to run before `pack`

Reply:
I've added a `BeforeTargets`. The Build target already depends on `AfterBuild`, which would cause a circular dependency if added too.